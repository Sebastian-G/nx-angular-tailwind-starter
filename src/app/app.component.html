<div class='bg-slate-100 h-full w-full text-center'>
  {{title}}
</div>
