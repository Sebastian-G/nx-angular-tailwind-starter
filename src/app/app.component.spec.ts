import { AppComponent } from './app.component';
import { MockBuilder, MockedComponentFixture, MockRender } from 'ng-mocks';
import { Component } from '@angular/core';

describe('AppComponent', () => {
  let fixture: MockedComponentFixture<AppComponent>;
  let component: AppComponent;

  beforeEach(() => MockBuilder(AppComponent));

  it('should create the app', () => {
    fixture = MockRender(AppComponent);
    component = fixture.point.componentInstance;
    fixture.detectChanges();
    expect(component).toBeTruthy();
  });

  it(`should have as title 'nt-angular-starter'`, () => {
    fixture = MockRender(AppComponent);
    component = fixture.point.componentInstance;
    fixture.detectChanges();
    expect(component.title).toEqual('nt-angular-starter');
  });

  it('should match snapshot', () => expect(MockRender(AppComponent).point.nativeElement).toMatchSnapshot());
});
