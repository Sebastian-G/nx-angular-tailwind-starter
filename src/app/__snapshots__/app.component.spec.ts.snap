// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppComponent should match snapshot 1`] = `
<app-root>
  <h1
    class="text-3xl font-bold underline"
  >
     Hello nt-angular-starter

  </h1>
</app-root>
`;
